htslib/hts_os.h

```diff
@@ -26,14 +26,23 @@ DEALINGS IN THE SOFTWARE.  */
 #ifndef HTSLIB_HTS_OS_H
 #define HTSLIB_HTS_OS_H
 
+#include "hts_defs.h"
+
 #ifdef __cplusplus
 extern "C" {
 #endif
 
-extern void hts_srand48(long seed);
-extern double hts_erand48(unsigned short xseed[3]);
-extern double hts_drand48(void);
-extern long hts_lrand48(void);
+HTSLIB_EXPORT
```

Comment:
Do these methods have to be exported from both the `*.h` and the `*.c` files (`os/rand.c` and `hts_os.c`)?

Reply:
I think it's because `hts_os.c` doesn't include `htslib/hts_os.h` (and possibly shouldn't due to `#define`s in the latter file.